Basic/ClientDriven/Assets/Scripts/ServerIngredientSpawner.cs

```diff
@@ -26,9 +26,9 @@ public class ServerIngredientSpawner : NetworkBehaviour
     public override void OnNetworkSpawn()
     {
         base.OnNetworkSpawn();
+        enabled = IsServer;
```

Comment:
Why is this line added? Same for the `enabled = false;` line below

Reply:
Ah this and the line below was a fix I added to ensure this NetworkBehaviour:
* is disabled as soon as NetworkManager is not listening
* can function normally in the case that a client disconnects and wants to host a session again (it would be disabled and never re-enabled before)